src/Aquarius.ONE.ClientSDK/Enterprise/Authentication/AuthenticationApi.cs

```diff
@@ -34,6 +34,7 @@ public AuthenticationApi(PlatformEnvironment environment, bool continueOnCapture
         public Token Token { get; set; }
         public event EventHandler TokenExpired = delegate { };
         public string AuthenticationUrl { get => _environment?.AuthenticationUri?.AbsoluteUri; }
+        public TimeSpan HttpClientTimeout = TimeSpan.FromMinutes(10);

```

Comment:
Is 10 minutes too long for a request timeout? Seems like a bug with the endpoint if a request takes that long.

Reply:
Ah sorry. I thought the timeout was a new change.

Reply:
10 minutes does seem awfully long, but if this is settable now that would be good